cmd/storagenode-updater/main_test.go

```diff
@@ -35,6 +35,8 @@ const (
 )
 
 func TestAutoUpdater_unix(t *testing.T) {
+	t.Skip("TODO: the version server must listen on random port")
```

Comment:
With this we also skipping test for storage node updating.

Reply:
True, but this test also needs to be fixed to use a random port. We can partially enable it back in another PR when fixing the port issue.

Reply:
Ok